app/controllers/api/order_cycles_controller.rb

```diff
@@ -0,0 +1,59 @@
+require 'open_food_network/products_renderer'
+
+module Api
+  class OrderCyclesController < BaseController
```

Comment:
We need a new spec and some tests api/order_cycles_controller_spec

Reply:
I'm just finishing it off :smile: